heareval/tasks/util/luigi.py

```diff
@@ -42,7 +45,8 @@ def name(self):
         # return type(self).__name__
 
     def output(self):
-        f = os.path.join(self.task_subdir, f"{self.stage_number:02d}-{self.name}.done")
+        #Replace the name with task_id as it is unique at task and parameter level
+        f = os.path.join(self.task_subdir, f"{self.stage_number:02d}-{self.task_id}.done")
```

Comment:
Added task_id in place of the name. The task id uses the parameters from the task and hashes them together to get a unique string for task parameter combinations. This is to solve the problem of the reusability of the Luigi task.

Reply:
Nice one! I like this solution. might also be nice to have the name in there too, if not just to help with debugging -- as long as it doesn't make the file name too long.

Did you mean to remove the luigi.LocalTarget return type?

Reply:
The task_id has the name attached to it. It goes like `name_params_hash`.